build(deps): bump webpack from 5.73.0 to 5.76.1 (#1387)
redpanda-data/documentation: modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

== Limitations

=== Lowercase field names required

> You must use only lowercase field names. AWS Glue automatically converts all table column names to lowercase. Redpanda matches columns by name for schema management, and using uppercase letters can cause errors when Redpanda can't find matching columns.

Reviewer: Replace can cause errors... with breaks schema management (end of sentence).

It is certain that it will break.

> You must use only lowercase field names. AWS Glue automatically converts all table column names to lowercase. Redpanda relies on exact column name matching to manage schemas, and using uppercase letters can break schema management when Redpanda can't find matching columns.

Reviewer 2: ```suggestion
Use only lowercase field names. AWS Glue converts all table column names to lowercase, and Redpanda requires exact column name matches for schema management. Using uppercase letters will prevent Redpanda from finding matching columns and break schema management.
```

Slight change to enforce active voice. Feel free to reject parts that you don't agree
